src/main/resources/issues_conditions.json

```diff
@@ -0,0 +1,7 @@
+###vms
+$.ManageIQ::Providers::Vmware::InfraManager[*].vms[?(@.host == null )].id
+$.ManageIQ::Providers::Vmware::InfraManager[*].vms[?(@.cpu_total_cores == 0 )].id
+$.ManageIQ::Providers::Vmware::InfraManager[*].vms[?(@.ems_ref == null )].id
+###hosts
+$.ManageIQ::Providers::Vmware::InfraManager[*].hosts[?(@.ems_cluster == null )].ems_ref
+###clusters
```

Comment:
The extension of the file is `.json` but the content is not a JSON object. Is it possible to use another type of extension for this file? just to be consistent between the filename and the content.

Reply:
I've renamed this to be .txt file